archey/logos/darwin.py

```diff
@@ -51,3 +51,50 @@
     """{c[0]}      {c[6]}################{c[0]}   """,
     """{c[0]}       {c[6]}####{c[0]}     {c[6]}#####{c[0]}    """,
 ]
+
+LOGO_WWDC = [
```

Comment:
Hi @twang817 ! Thanks for your PR.

Some thoughts :

* your ASCII art is **really** big (despite being pretty though 😇) : it weighs around 21KB, which is disproportionate according to others. Do you think you could rework it to get closer to the "classic" dimensions ?

Reply:
I didn't draw this. I came across the wwdc logo and copied it from there. 

https://www.reddit.com/r/apple/comments/4ob7q2/anyone_else_love_the_asciistyle_apple_logo_from/?utm_source=share&utm_medium=ios_app&utm_name=iossmf

I wouldn't know where to begin with shrinking this.